addons/mail/static/src/discuss/core/web/discuss_tour.js

```diff
@@ -87,6 +87,7 @@ registry.category("web_tour.tours").add("mail_tour", {
                 )
             ),
             position: "bottom",
+            isCheck: true,
```

Comment:
See review point `4.`, don't change this step but add a new one (check:true) for test only (auto:true)